web/pdf_page_view.js

```diff
@@ -583,7 +745,11 @@ class PDFPageView {
 
     if (!keepCanvasWrapper && this.#canvasWrapper) {
       this.#canvasWrapper = null;
-      this.#resetCanvas();
+      this._resetCanvas();
+    }
+
+    if (!preserveDetailViewState) {
+      this.detailView?.reset({ keepCanvas: keepCanvasWrapper });
```

Comment:
When we're doing a "full" reset, i.e. calling `this.reset()` without any parameters (e.g. what happens when evicting pages from the `PDFPageViewBuffer`-instance), shouldn't we also set `this.detailView = null;` to clear it out?